cli/src/Morphir/Web/DevelopApp.elm

```diff
@@ -1717,12 +1742,12 @@ viewAsLabel clickMessage theme header class backgroundColor docs content =
         ]
         [ el
             [ Font.bold
-            , paddingXY (theme |> Theme.scaled 0) (theme |> Theme.scaled -2)
+            , paddingXY 1 (theme |> Theme.scaled -3)
             ]
             header
         , el
             [ alignRight
-            , paddingXY (theme |> Theme.scaled 0) (theme |> Theme.scaled -2)
+            , paddingXY 1 (theme |> Theme.scaled -3)
```

Comment:
We should use theme controlled sizes everywhere. Here you have a mix of static and theme controlled size. The approach with `scaled` was just to keep it flexible but we might be able to pre-define some specific gaps like `smallGap`, `largeGap` or similar and use those instead of these weird numbers.